tools/docs/generate_external_deps_rst.py

```diff
@@ -126,7 +126,14 @@ def main():
         name = rst_link(project_name, project_url)
         version = rst_link(render_version(v['version']), get_version_url(v))
         release_date = v['release_date']
-        dep = Dep(name, project_name.lower(), version, cpe, release_date)
+        license = v.get('license', '')
+        if license == 'N/A':
```

Comment:
im wondering if we need to accomodate N/A if the license is not compulsory (perhaps it should be)

Reply:
I am not sure if license is compulsory or not, but in my opinion, I think license should have the same priority as CVE. In fact, CVE is also not compulsory at the moment. It is a simple copy-paste, and I can change it if needed.

Reply:
yep - lets go with not making it compulsory, at least for now - but lets drop the N/A - rather if its not filled in just dont add it

Reply:
Fixed.